docs/home/howto/UPGRADE.md

```diff
@@ -124,6 +124,7 @@ The `epicli upgrade` command has additional flags:
 ###  Run *apply* after *upgrade*
 
 Currently Epiphany does not fully support apply after upgrade. There is a possibility to re-apply configuration from newer version of Epicli but this needs some manual work from Administrator. Re-apply on already upgraded cluster needs to be called with `--no-infra` option to skip Terraform part of configuration.
+If apply after upgrade is run with `--no-infra` the used system images from the older Epiphany version are preserved to prevent the destruction of the VM`s.
```

Comment:
```suggestion
If `apply` after `upgrade` is run with `--no-infra`,  the used system images from the older Epiphany version are preserved to prevent the destruction of the VMs.
```